frontend/packages/client/src/utils.js

```diff
@@ -184,3 +184,10 @@ export const customDraftToHTML = (content) => {
 };
 
 export const isValidAddress = (addr) => /0[x,X][a-zA-Z0-9]{16}$/gim.test(addr);
+
+export const wait = async (milliSeconds = 5000) =>
+  new Promise((resolve) => {
+    setTimeout(() => {
+      resolve(true);
+    }, milliSeconds);
+  });
```

Comment:
this is not used anywhere in the production code, but it's used when trying to create a delay on loading states.